contracts/strategies/qv-hackathon/HackathonQVStrategy.sol

```diff
@@ -0,0 +1,269 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.19;
+
+import {IAllo} from "../../core/IAllo.sol";
+import {IRegistry} from "../../core/IRegistry.sol";
+import {QVSimpleStrategy} from "../qv-simple/QVSimpleStrategy.sol";
+import {Metadata} from "../../core/libraries/Metadata.sol";
+
+import {EnumerableSet} from "@openzeppelin/utils/structs/EnumerableSet.sol";
+
+// Note: EAS Contracts
+import {
+    AttestationRequest,
+    AttestationRequestData,
+    IEAS,
+    Attestation,
+    MultiAttestationRequest,
+    MultiRevocationRequest
+} from "@ethereum-attestation-service/IEAS.sol";
+import {ISchemaRegistry, ISchemaResolver, SchemaRecord} from "@ethereum-attestation-service/ISchemaRegistry.sol";
+
+contract HackathonQVStrategy is QVSimpleStrategy {
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    error ALREADY_ADDED();
+
+    // The instance of the EAS and SchemaRegistry contracts.
+    IEAS public eas;
+    ISchemaRegistry public schemaRegistry;
+
+    bytes32 constant EMPTY_UID = 0;
+
+    // Who can call submitAttestation
+    mapping(address => bool) public attestationSigners;
```

Comment:
aren't `attestationSigners` equal to `poolManager`?
There is no real distinction on this smart contract, since `poolManager` can attest and in some functions you have to be a `poolManager` and a `attestationSigners`

i think `attestationSigners` can be removed, or?

Reply:
@nfrgosselin 
any opinions on dedicated attestation signer?